demo-shell/src/app.config.json

```diff
@@ -1197,5 +1197,178 @@
     {
       "name": "subprocessapp"
     }
-  ]
+  ],
+  "adf-mime-types": [
```

Comment:
please move in a ts file, I don't think for now we want to make it configurable , remove also from the schema for now